app/client/sass/tablet_view.scss

```diff
@@ -94,31 +132,40 @@
     margin: 0.25em;
     display: inline-block;
     float: right;
+    clear: both;
     font-size: 1.15em;
   }
 
   .job-information {
     section {
-      display: flex;
+      aside {
+        margin: 0em;
+        padding: 0em 1em;
+      }
     }
   }
 
   .thumbnail {
-    margin: 1.5em;
-    height: 200px;
-    width: 200px;
+    height: 240px;
+    width: 240px;
   }
 
   .walk-score-information {
-    width: 40%;
-    display: flex;
-    float: right;
-    position: relative;
-    top: -125px;
+    p {
+      padding: 2em;
+      font-size: 2em;
+    }
   }
 
   .date-posted {
-    position: relative;
-    top: -125px;
+    font-size: 2em;
+  }
+
+  .company-name {
+    font-size: 2em;
+  }
+
+  #job-title {
```

Comment:
This is the one ID in the project. It was not working with just a class and I am not sure why...hence the ID.